pkg/reconciler/instances/ory/test/ory_integration_test.go

```diff
@@ -0,0 +1,63 @@
+package test
+
+import (
+	"context"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/kyma-incubator/reconciler/pkg/logger"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/kubernetes"
+	"github.com/kyma-incubator/reconciler/pkg/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubectl/pkg/util/podutils"
+)
+
+const (
+	envOryIntegrationTests = "ORY_RECONCILER_INTEGRATION_TESTS"
```

Comment:
This constant only refers to the key of the env, not the value, so maybe you could include `key` keyword here?

Reply:
I would leave it as env, for the environment variable, so the name ORY_RECONCILER_INTEGRATION_TESTS fits here.